packages/web/src/components/comments/CommentForm.tsx

```diff
@@ -24,6 +27,22 @@ type CommentFormProps = {
   isLoading?: boolean
 }
 
+// This is annoying af to have to make a component for; but necessary so that can use the resetForm method from context
+const FormResetHandler = ({
+  isLoading
+}: {
+  isLoading: boolean | undefined
+}) => {
+  const prevIsLoading = usePrevious(isLoading)
+  const { resetForm } = useFormikContext()
+  useEffect(() => {
+    if (!isLoading && prevIsLoading) {
+      resetForm()
+    }
+  }, [prevIsLoading, isLoading, resetForm])
+  return null
+}
```

Comment:
This is annoying 🤮 
Formik context is really gross sometimes;

Reply:
oh weird

Reply:
It's not too bad, but you could also split the whole form content into a separate component and include this hook in there

Reply:
Yeah its not the worst tbh but it just _feels_ like it should be easier to manage via the parent component; feels like it should be easier to access the form from the parent tbh